org.scram_pra.scram.yml

```diff
@@ -0,0 +1,42 @@
+app-id: org.scram_pra.scram
+runtime: org.kde.Sdk
+runtime-version: 5.15-21.08
+sdk: org.kde.Sdk
+command: scram-gui
+rename-icon: scram
+rename-desktop-file: scram-gui.desktop
```

Comment:
You also need `rename-appdata-file: scram-gui.appdata.xml`

Reply:
done